### DISCUSSION TITLE
Chatgpt plugin for Trilium

### DISCUSSION THREAD

**Discussion by soulsands** [TriliumNext/Trilium#4007] [Show and tell]
Hello everyone, I have created a chat plugin. It's not yet fully developed, but it's already quite useful. If you find any problems or have any suggestions, please let me know. **Any help** would be appreciated!

https://github.com/soulsands/trilium-chat

**Comment by Nriver** [comment]
Looks awesome!

**Comment by walker83** [comment]
It's really cool, I like it very much.
